api/commands/compile.ts

```diff
@@ -65,19 +65,17 @@ export async function compile(
 
 export class CompileChildProcess {
   public static forkProcess() {
-    // Runs the worker_bundle script we generate for the package (see packages/@dataform/cli/BUILD)
-    // if it exists, otherwise run the bazel compile loader target.
-    const findForkScript = () => {
-      try {
-        const workerBundlePath = require.resolve("./worker_bundle");
-        return workerBundlePath;
-      } catch (e) {
-        return require.resolve("../../sandbox/vm/compile_loader");
-      }
-    };
-    const forkScript = findForkScript();
+    let workerBundle: string;
+    try {
+      // The bundled CLI packages the worker_bundle directly.
+      workerBundle = require.resolve("./worker_bundle");
```

Comment:
Given your changes now, I suspect you can get rid of the try catch and just use this line for both cases? `resolve("./worker_bundle")` should work in both scenarios!

Reply:
I think it's neater (/more bazely) to have this than have multiple genrules copying the bundle to the correct places in order to replicate the CLI bundle locality